.travis.yml

```diff
@@ -9,29 +31,12 @@ notifications:
 env:
   global:
     - secure: "wnthDxOwv7qaVfJQfwYQLFo4DDBgfT4HoLntBsfmYjwmSVzCuXPktCK3vOpimb5twyiVNkt1I/82x8lyYpiczKqPcNZqd8p91mInt5eKIf8Gacqb3a1FwGyk9bwHSZdYpQiblY/wMVb0+WJTuhlVf1NI5xXMWCf1l88+1q6E6jo="
```

Comment:
this isn't the most secure thing to do, no? See http://docs.travis-ci.com/user/environment-variables/